react/features/base/jwt/functions.ts

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line import/order
```

Comment:
Why is this needed? It's properly sorted...

Reply:
Because we've got a conflict with the comment below, it requires a space that triggers this error.